docs/make.bat

```diff
@@ -0,0 +1,263 @@
+@ECHO OFF

```

Comment:
Ok, I am happy now, but we must remove the .bat file. We shouldn't pretend that we support it.